charts/gateway/README.md

```diff
@@ -12,22 +12,38 @@ A Helm chart for Trino Gateway
 * <https://github.com/trinodb/trino-gateway>
 
 ## Values
-* `replicaCount` - int, default: `1`
-* `image.repository` - string, default: `"trinodb/trino-gateway"`  
```

Comment:
There are many changes in the README that do not appear to be related to the changes in your values.yaml. Please revert and then use `precommit` (see instructions in base README) to make sure the README is updated appropriately